scripts/sitemap.py

```
@@ -0,0 +1,110 @@
+            changefreq.text = url_data['changefreq']
+
+            priority = SubElement(url, 'priority')
+            priority.text = str(url_data['priority'])
+
+        # 美化输出
+        rough_string = tostring(urlset, encoding='unicode')
+        reparsed = minidom.parseString(rough_string)
+        return reparsed.toprettyxml(indent="  ")
+
+    def generate_json(self) -> str:
+        """生成 JSON 格式（供前端使用）"""
```

Ruff (0.15.9)

[warning] Docstring contains ambiguous `（` (FULLWIDTH LEFT PARENTHESIS). Did you mean `(` (LEFT PARENTHESIS)?

(RUF002)